scripts/st2bootstrap-deb.sh

```diff
@@ -79,6 +79,10 @@ install_st2mistral() {
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
   # Register mistral actions
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf populate
+  
+  # Start Mistral
+  sudo service mistral-server start
+  sudo service mistral-api start
 }
```

Comment:
2) I think your concern is st2ctl won't work here because it already started st2 services in previous step. For already started services, service foo start is a noop. So it should be fine to do st2ctl start although i prefer this.